config/mcus/octopus.cfg

```diff
@@ -1,59 +1,65 @@
-    DRIVER7_STEP=PE6  , DRIVER7_DIR=PA14 , DRIVER7_ENABLE=PE0  , DRIVER7_TMCUART=PD3  ,
-
-[board_pins endstops]
-mcu: mcu
-aliases:
-    STOP0=PG6, STOP1=PG9, STOP2=PG10, STOP3=PG11,
-    STOP4=PG12, STOP5=PG13, STOP6=PG14, STOP7=PG15,
-
-[board_pins extruder]
-mcu: mcu
-aliases:
-    HE0=PA2, HE1=PA3, HE2=PB10, HE3=PB11,
-
-[board_pins bed]
-mcu: mcu
-aliases:
-    BED0=PA1,
-
-[board_pins termistors]
-mcu: mcu
-aliases:
-    TB=PF3, T0=PF4, T1=PF5, T2=PF6, T3=PF7
-
-[board_pins expansion]
-mcu: mcu
-aliases:
-# EXP1 / EXP2 (display) pins
-    # EXP1 header
-    EXP1_1=PE8, EXP1_2=PE7,
-    EXP1_3=PE9, EXP1_4=PE10,
-    EXP1_5=PE12, EXP1_6=PE13,    # Slot in the socket on this side
-    EXP1_7=PE14, EXP1_8=PE15,
-    EXP1_9=<GND>, EXP1_10=<5V>,
-
-    # EXP2 header
-    EXP2_1=PA6, EXP2_2=PA5,
-    EXP2_3=PB1, EXP2_4=PA4,
-    EXP2_5=PB2, EXP2_6=PA7,      # Slot in the socket on this side
-    EXP2_7=PC15, EXP2_8=<RST>,
-    EXP2_9=<GND>, EXP2_10=<5V>,
-
+[mcu]
+##--------------------------------------------------------------------
+# This board work by using a serial connection by default.
+# If you want to use CAN, invert the commented lines here
+
+# canbus_uuid: change-me-to-the-correct-canbus-id
+# serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+##--------------------------------------------------------------------
+
+[board_pins drivers]
+mcu: mcu
+aliases:
+# Motor driver pins
+    DRIVER0_STEP=PF13 , DRIVER0_DIR=PF12 , DRIVER0_ENABLE=PF14 , DRIVER0_TMCUART=PC4  ,
+    DRIVER1_STEP=PG0  , DRIVER1_DIR=PG1  , DRIVER1_ENABLE=PF15 , DRIVER1_TMCUART=PD11 ,
+    DRIVER4_STEP=PF9  , DRIVER4_DIR=PF10 , DRIVER4_ENABLE=PG2  , DRIVER4_TMCUART=PF2  ,
+    DRIVER5_STEP=PC13 , DRIVER5_DIR=PF0  , DRIVER5_ENABLE=PF1  , DRIVER5_TMCUART=PE4  ,
+    DRIVER6_STEP=PE2  , DRIVER6_DIR=PE3  , DRIVER6_ENABLE=PD4  , DRIVER6_TMCUART=PE1  ,
+    DRIVER7_STEP=PE6  , DRIVER7_DIR=PA14 , DRIVER7_ENABLE=PE0  , DRIVER7_TMCUART=PD3  ,
+
+[board_pins endstops]
+mcu: mcu
+aliases:
+    STOP0=PG6, STOP1=PG9, STOP2=PG10, STOP3=PG11,
+    STOP4=PG12, STOP5=PG13, STOP6=PG14, STOP7=PG15,
+
+[board_pins extruder]
+mcu: mcu
+aliases:
+    HE0=PA2, HE1=PA3, HE2=PB10, HE3=PB11,
+
+[board_pins bed]
+mcu: mcu
+aliases:
+    BED0=PA1,
+
+[board_pins termistors]
+mcu: mcu
+aliases:
+    TB=PF3, T0=PF4, T1=PF5, T2=PF6, T3=PF7
+
+[board_pins fans]
+mcu: mcu
+aliases:
+    # FAN6 and FAN7 are not controllable
+    FAN0=PA8, FAN1=PE5, FAN2=PD12, FAN3=PD13, FAN4=PD14, FAN5=PD15
+
+[board_pins expansion]
+mcu: mcu
+aliases:
+# EXP1 / EXP2 (display) pins
+    # EXP1 header
+    EXP1_1=PE8, EXP1_2=PE7,
+    EXP1_3=PE9, EXP1_4=PE10,
+    EXP1_5=PE12, EXP1_6=PE13,    # Slot in the socket on this side
+    EXP1_7=PE14, EXP1_8=PE15,
+    EXP1_9=<GND>, EXP1_10=<5V>,
+
+    # EXP2 header
+    EXP2_1=PA6, EXP2_2=PA5,
+    EXP2_3=PB1, EXP2_4=PA4,
+    EXP2_5=PB2, EXP2_6=PA7,      # Slot in the socket on this side
+    EXP2_7=PC15, EXP2_8=<RST>,
+    EXP2_9=<GND>, EXP2_10=<5V>,
+
```

Comment:
I think you made a mistake on this file as everything looks to be changed even if just a couple of things were added. Maybe the line endings changed ? Can you check and correct it ?

Reply:
it's not a mistake, all EOL on this file were DOS, I've converted it to UNIX (and added more pins)

Reply:
Ohhh, this make sense. Thanks for converting it to proper lines ending then :)